orc8r/cloud/go/services/analytics/analytics/main.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defaultAnalysisSchedule = "*/1 * * * *" // Every 1 minute
+	defaultAnalysisSchedule = "*/15 * * * *" // Every 15 minutes
 )
 
 //ServiceMain - entrypoint for analytics service
```

Comment:
Additional comments for main
- the flag.Parse is redundant, since flag.Parse is done in NewOrchestratorService
- here and elsewhere: logs should start with capital letter, errors should start with lowercase (except for good reason)
- `Analytics Service Config` -> `Analytics service config` as there's no reason to capitalize the subsequent words
- Just call it `getExporter`, the `IfRequired` decreases readability and doesn't add much. You can add the "if required" in a doc comment if desired
- Instead of glog.Info+return, just use glog.Fatal